addons/website_hr_recruitment/security/ir.model.access.csv

```diff
@@ -1,3 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_hr_job_public,hr.job.public,hr.model_hr_job,,1,0,0,0
-access_hr_department_public,hr.department.public,hr.model_hr_department,base.group_public,1,0,0,0
+access_hr_department_public,hr.department.public,hr.model_hr_department,,1,0,0,0
```

Comment:
@odony Hello, I would like your opinion here.

What I would generally do in such cases in stable, is to see and try to use a `sudo()` somewhere rather than changing security rules (which I would leave for master).
But in this case, I can't see any issue, even considering possible customization, as the public user can read a record that a portal user can't :grimacing: 
Am I missing something obvious or is this fine?
Arguably, to fix it without the need of a `-u`, we could also add a `sudo()` in the `hr_department` `name_get()` method which is raising the 403 but I doubt it will be enough. It will most likely fail later.

Reply:
Thanks @rdeodoo :)

I wouldn't do the `sudo`, as I think it is easy enough to manually remove the group on the ACL honestly.

Reply:
I am just being cautious, there is probably nothing wrong here since that is just basically adding the public right to the other users.
But since that's a change in a security file in a ~3 year old stable version, better safe than sorry.
I can't imagine how that would be problematic tho, indeed.

> I wouldn't do the sudo, as I think it is easy enough to manually remove the group on the ACL honestly.

Yes but that's not helping anyone if they don't do a module update. Which is fine to me, as this is only reported now, probably somehow not a common case.

Reply:
Hello @odoo/rd-security 

Could someone check if this is OK from a security point of view?
Thanks!

Reply:
Usually I would argue for the sudo but here we have the route `'/jobs/department/<model("hr.department"):department>'` that is an issue.
A potential issue is that portal has RPC capabilities (while public does not). There is nothing very sensitive in our department fields (well there are private computed fields but you need higer access rights) but nothing guarantee it's not the case in another module. The issue is that hr.department does not "sound" like a public model (unlike an event or blog post).
I _think_ it's safe here but could be nice if we can remove it in an higher version.